src/main/resources/files/gradle/build.gradle.tpl

```diff
@@ -15,9 +15,17 @@ repositories {
 }
 
 dependencies {
-    providedCompile 'org.eclipse.microprofile:microprofile:[# th:text="${mp_version}"/]'
+    [# th:if="${mp_version} eq '4.0'"]
+    providedCompile 'org.eclipse.microprofile:microprofile:[# th:text="4.0.1"/]'
```

Comment:
We have this kind of config already in MicroProfileVersion https://github.com/eclipse/microprofile-starter/blob/2c4847e854dfcadaf2b6126638acbf825672ae57/src/main/java/org/eclipse/microprofile/starter/core/model/MicroProfileVersion.java#L30

So we should define a new variable with the `mavenVersion` content and use this.

Reply:
Good point! Thanks @rdebusscher ! I have done the changes as suggested.